common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4864,6 +4866,23 @@ object CelebornConf extends Logging {
       .booleanConf
       .createWithDefault(false)
 
+  val CLIENT_CHUNK_PREFETCH_ENABLED: ConfigEntry[Boolean] =
+    buildConf("celeborn.client.chunk.prefetch.enabled")
+      .categories("client")
+      .doc("Whether to enable chunk prefetch when creating CelebornInputStream.")
+      .version("0.6.0")
+      .booleanConf
+      .createWithDefault(false)
+
+  val CLIENT_INPUTSTREAM_CREATION_WINDOW: ConfigEntry[Int] =
+    buildConf("celeborn.client.inputStream.creation.window")
+      .categories("client")
+      .doc(s"Window size that CelebornShuffleReader pre-creates CelebornInputStreams, for coalesced scenario" +
```

Comment:
Although this pr is suitable for coalesced scenarios, this is no limit for a task that reads multiple locations. I think this might cause unexpected high memory consumption.